### ISSUE TITLE
Excalidraw scrappy font is hard to read

### ISSUE THREAD

**Issue by sompylasar** [excalidraw/excalidraw#1677]
Maybe it's easy for some people, but for me personally it's so hard to read that I always switch to Code font in my diagrams (thank you for this option!).

I understand it has to do with the scrappiness of the drawing, but can this scrappy font be more legible please? (I'm not asking to remove the scrappiness, rather tweak the scrappy font to also make it more legible.)

I'm struggling with:
- Low contrast between capital and lower case letters.
- The digit "1" – it is just a stick, very similar to lowercase "L" and uppercase "i"
- Little spacing between some letters make them glue together like ligatures (which I find harder to parse).

**Comment by TimoSta** [comment]
I see the same issue as you.

#1651 would resolve this!

**Comment by Yourownfont** [comment]
Constructive comments thank you - but hard to work with. Do you have any visual examples of your issues? I am working with the font right now so just add them and I'll change them as they come!

Thanks,
Ellinor

**Comment by sompylasar** [comment]
Thank you Ellinor for looking into this. I cannot share specific diagrams and text from them that I found took too much time to parse, but I tried to come up with similar combinations of letters in the image below. I noticed that the readability is highly dependent on the familiarity with the specific terms. I wrote this feedback from looking at a large diagram with unfamiliar concepts for the first time (also English is not my first language). Now that I look at it again, it's still not a snap of fingers to read a word but much easier after I've already seen these words written in this font.

![Untitled-2020-06-03-1312](https://user-images.githubusercontent.com/498274/83686617-5dc81c00-a59f-11ea-999a-4f5bfb544e1f.png)

**Comment by thorn0** [comment]
> Low contrast between capital and lower case letters.

I disagree on this point for Latin letters, but this is pretty much the case for Cyrillic.

> The digit "1" – it is just a stick

Agree, it's not very readable on its own in captions like "Group 1". Captions like "Group 10" or "2019" are fine though.

**Comment by cnscorpions** [comment]
pls consider adding font selection functionality. :)

**Comment by Jan-M** [comment]
`88.9` is also very unreadable, more though when zooming.

![image](https://user-images.githubusercontent.com/1443395/90014425-52802380-dca7-11ea-877e-491074d16d45.png)

**Comment by dwelle** [comment]
One more :)

![image](https://user-images.githubusercontent.com/5153846/90014829-f9fd5600-dca7-11ea-8adf-1d690f523ca6.png)

**Comment by vjeux** [comment]
We're looking for another handwritten font. If you could add suggestions for the fonts listed in this issue that'd be useful. Thanks! #2945

**Comment by VaughnVernon** [comment]
FWIW, I like your font, expect the capital D is bothersome (over-the-top arch lines to the left). I think a few other characters are, too.

I have used HANDYHANDY and Drafting Table. There are a few others that are nice, and I reference Andre Drafting below. I have no idea of cost/royalties for massively internet services/applications. I think some are "free" as in donations can be given. Maybe OSS and free services get special consideration.

https://www.dafont.com/handyhandy.font

https://www.dafont.com/drafting-table.font

https://www.myfonts.com/fonts/andre-design/andre-drafting/
